webpack.config.js

```diff
@@ -0,0 +1,60 @@
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+
```

Comment:
Please take a look on CopyWebpackPlugin and EslintWebpackPlugin.
CopyWebpackPlugin is needed for transfering existing files for build process
EslintWebpackPlugin could be useful for lint-checking during development